compiler-plugin/testData/api/referenceElement.kt

```diff
@@ -22,16 +22,27 @@
 // KSClassifierReferenceImpl: Qualifier of C is A
 // KSClassifierReferenceImpl: Qualifier of Int is null
 // KSClassifierReferenceImpl: Qualifier of String is null
-// KSClassifierReferenceDescriptorImpl: Qualifier of B is A
-// KSClassifierReferenceDescriptorImpl: Qualifier of C<Int> is A<String>
```

Comment:
is the removal of these 2 lines expected?

Reply:
Yes. Previously, they were created from source file. Now they are moved to lib module and renamed.